README.md

```diff
@@ -71,19 +79,19 @@ https://github.com/foostan/crkbd/
 
 
 ## Zen
-https://www.rgbkb.net/products/zen-keyboard  
-https://mykeyboard.eu/catalogue/category/keyboard-kits/zen-keyboard_55/  
+https://www.rgbkb.net/products/zen-keyboard
+https://mykeyboard.eu/catalogue/category/keyboard-kits/zen-keyboard_55/
 
-https://legonut.gitbooks.io/zen-keyboard/content/  
-https://legonut.gitbooks.io/zen-keyboard/content/flashing.html  
-https://github.com/qmk/qmk_firmware/blob/master/keyboards/zen/readme.md  
+https://legonut.gitbooks.io/zen-keyboard/content/
```

Comment:
Oof! Why all the whitespace removals in this PR.. Let's keep it focused on just your additions if that's alright.